docs/docs/Integrations/Arize/integrations-arize.md

```diff
@@ -19,19 +19,32 @@ To add tracing to your Langflow application, add the `ARIZE_SPACE_ID` and `ARIZE
 3. Copy the **SpaceID** and **API Key (Ingestion Service Account Key)** values.
 4. Create a `.env` file in the root of your Langflow application.
 5. Add the `ARIZE_SPACE_ID` and `ARIZE_API_KEY` environment variables to your Langflow application.
-You do not need to specify the **Arize Project** name if you're using the standard Arize platform. The **Project** name in Arize is the same as the Langflow **Flow** name.
+You do not need to specify the **Arize Project** name if you're using the standard Arize platform.
+Replace the following:
+
+* YOUR_ARIZE_SPACE_ID: the **SpaceID** value copied from Arize
+* YOUR_ARIZE_API_KEY: the **API Key** value copied from Arize
 
 ```bash
-export ARIZE_SPACE_ID=<your-arize-space-id>
-export ARIZE_API_KEY=<your-arize-api-key>
+ARIZE_SPACE_ID=YOUR_ARIZE_SPACE_ID
+ARIZE_API_KEY=YOUR_ARIZE_API_KEY
 ```
 
-6. Start your Langflow application with the values from the `.env` file.
+6. Optional: if you are using the [Arize Phoenix platform](https://docs.arize.com/phoenix), add the `PHOENIX_API_KEY` environment variable to your application instead of `ARIZE_SPACE_ID` and `ARIZE_API_KEY`.
```

Comment:
If you are using the Phoenix platform instead of the standard Arize platform, this step is too late because you've already followed step 5 to use the space ID and API key values. Please reworks steps 5 and 6.

Reply:
Added tabs